tests/geometry/WorkingGeometry.spec.ts

```diff
@@ -58,16 +59,97 @@ describe('WorkingGeometry', () => {
                 vec3.fromValues(1, 1, 0),
                 vec3.fromValues(1, 0, 0)
             ];
-            const normal = vec3.fromValues(0, 1, 0);
-            const faces = [new Face([0, 1, 2], normal), new Face([1, 2, 3], normal)];
+            const normal = vec3.fromValues(0, 0, 1);
+            const faces = [new Face([0, 1, 2], normal), new Face([0, 2, 3], normal)];
             const controlPoints = [vec3.fromValues(0, 0, 0)];
             const square = new WorkingGeometry(vertices, faces, controlPoints);
             const bakedSquare = square.bake();
 
-            // Not testing the colors yet since they don't do anything useful.
-            expect(bakedSquare.indices).toEqual([0, 1, 2, 1, 2, 3]);
+            // Not testing the colors yet since they don't do anything useful
+            expect(bakedSquare.indices).toEqual([0, 1, 2, 0, 2, 3]);
             expect(bakedSquare.vertices).toEqual(vertices);
             expect(bakedSquare.normals).toEqual([normal, normal]);
         });
+        it('can bake a WorkingGeometry that has many merged objects', () => {
+            const rootSquare = TestHelper.square();
+            const childSquare1 = TestHelper.square(vec3.fromValues(1, 0, 0));
+            const childSquare2 = TestHelper.square(vec3.fromValues(-1, 0, 0));
+            const grandChildSquare = TestHelper.square(vec3.fromValues(1, 0, 1));
+            rootSquare.merge(childSquare1);
+            rootSquare.merge(childSquare2);
+            childSquare1.merge(grandChildSquare);
+
+            const bakedObject = rootSquare.bake();
+            expect(bakedObject.indices).toEqual([
+                0,
```

Comment:
So I had all of these number nicely grouped into groups of 3... but `yarn fix-format` didn't like that very much. Unsure if there's a way to fix that. Open for suggestions.

Reply:
I think if you put a comment between them it will force them to stay on separate lines? not ideal but maybe worth the benefits of autoformatting overall. Not the end of the world if it's just all separate though

Reply:
Alternatively, maybe we should update the interface so that indices are also a `vec3` array, given how they always come in threes? Don't let that block this PR, but if you think it'll make the interface easier to reason about in general, it could be worth doing later